lib/oauth2/error.rb

```diff
@@ -16,7 +16,8 @@ def initialize(response)
         message << "#{@code}: #{@description}"
       end
 
-      message << response.body
+      #https://github.com/intridea/oauth2/issues/155
+      message << response.body.force_encoding("UTF-8")
```

Comment:
I think I would prefer to get this out into production sooner than later to see what the error is before we go nuts on this fix